api/models/Events.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+
+var LinkingModels = require('sails-linking-models');
+var ModelHelpers = require('../lib/ModelHelpers');
+
+const model = {
+  schema: true,
+  autoCreatedAt: true,
+  autoUpdatedAt: true,
+  autoPK: false,
+  attributes: {
+    operation: {
+      type: 'string',
+      required: true,
+      enum: ['check in']
+    },
+    event_received: {
```

Comment:
the date + time we receive the event will be stored in the auto `created` but we don't really care about that, what we want to know is the date it occurred, taking the date from what is given to us in the event payload itself